### PR TITLE
Cybersource REST: Adding capture request

### PR DESCRIPTION
**Pull request by Heavyblade**
## Summary:

Adding the capture functionality to the Cybersource REST gateway

### Remote Test:
Finished in 25.504733 seconds.
25 tests, 89 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### Unit Tests:

Finished in 39.743032 seconds.
5468 tests, 77209 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### RuboCop:
760 files inspected, no offenses detected

### CODE REVIEW

**Review by jherreraa — APPROVED** [review]
LGTM 👍

File: lib/active_merchant/billing/gateways/cyber_source_rest.rb [hunk 1]
```diff
@@ -51,6 +51,13 @@ def authorize(money, payment, options = {}, capture = false)
         commit('payments', post)
       end
 
+      def capture(money, authorization, options = {})
+        payment = authorization.split('|').first
+        post = build_refund_request(money, options)
```
Comment (curiousepic) [inline lib/active_merchant/billing/gateways/cyber_source_rest.rb:56]: let's rename `build_refund_request` to something more generic too prevent confusion - seems weird we are using it for captures. maybe something like `build_reference_request`

Comment (Heavyblade) [reply lib/active_merchant/billing/gateways/cyber_source_rest.rb]: you are right @curiousepic, change done

**Review by curiousepic — APPROVED** [review]
Thanks, 👍

**Pull request closed without merging**